### DISCUSSION TITLE
Q: Does neovim support sysinit.lua? Should it?

### DISCUSSION THREAD

**Discussion by LunarLambda** [neovim/neovim#30247] [General]
I know one can write
```vim
lua <<END
-- lua things
END
```
in `sysinit.vim` but this is a bit hacky. I found it surprising the docs don't mention an analog to init.lua for sysinit.

**Comment by dundargoc** [comment]
> Does neovim support sysinit.lua? 

No.

> Should it? 

Nah. You can just use sysinit.vim to call a chunk of lua code of your choosing.

Related: https://github.com/neovim/neovim/issues/22595

**Comment by clason** [comment]
Adding to that, system-wide configuration is a Vim anti-pattern we don't want to support; this has led to _way_ too many frustrations before.
